src/main/resources/templates/screens/admin/accounts/new.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html
+    xmlns:th="http://www.thymeleaf.org"
+    lang="en"
+    th:replace="~{/layouts/admin-layout :: admin-layout-fragment(
+        ~{::title},
+        ~{::#static-resources},
+        ~{::#main-content}
+    )}"
+>
+<head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <title>Create seller account</title>
+    <th:block id="static-resources">
+    </th:block>
+</head>
+<body id="main-content">
+<div class="container-fluid py-2">
```

Comment:
```suggestion
<div class="container py-2">
``` Đổi thành container thôi cho hắn đỡ rộng ái form